src/firstSuggestionHint.ts

```diff
@@ -0,0 +1,69 @@
+import { DecorationOptions, DecorationRangeBehavior, ExtensionContext, Position, Range, window } from "vscode";
```

Comment:
the terminology here is a bit off imo - all references like `displayFirstSuggetionHint` should be `displayFirstSuggetionDecoration`